src/components/custom/other-platforms.js

```diff
@@ -269,11 +265,28 @@ OtherPlatform.propTypes = {
 }
 
 export const NavPlatform = ({ onClick, is_ppc, is_ppc_redirect }) => {
+    const getDtraderText = () => (
+        <NavCard
+            aria_label="Dtrader"
+            icon={() => <img src={DTrader} alt="" width="32" height="32" />}
+            content={
+                <Localize translate_text="A whole new trading experience on a powerful yet easy to use platform." />
+            }
+            title={<Localize translate_text="DTrader" />}
+            onClick={onClick}
+            to="/dtrader/"
+        />
+    )
     return (
         <Flex>
             {!is_ppc && (
                 <>
-                    <Flex direction="column" wrap="wrap" jc="flex-start">
+                    <Flex
+                        direction="column"
+                        wrap="wrap"
+                        jc="flex-start"
+                        style={{ boxShadow: 'inset -1px 0px 0px #F2F3F4' }}
```

Comment:
You can use var(--color-grey-8) for this.

Reply:
Done